src/js/_enqueues/lib/comment-reply.js

```diff
@@ -396,6 +418,7 @@ window.addComment = ( function( window ) {
 		temporaryElement = document.createElement( 'div' );
 		temporaryElement.id = temporaryFormId;
 		temporaryElement.style.display = 'none';
+		temporaryElement.textContent = initialHeadingText;
```

Comment:
Does having this hidden text element on the page cause any issue with screen readers?

Reply:
No, it'll be invisible to screen readers because it's styled with `display: none`.